Library/Homebrew/formula.rb

```diff
@@ -2460,3 +2460,7 @@ def link_overwrite_paths
     end
   end
 end
+
+require "metaformula"
+# require metaformula so that whenever {Formula} is visible, so is {MetaFormula}
+# require in the end to avoid circular requirement
```

Comment:
That's still a circular requirement. I'd remove it here and add the require somewhere else.